apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/TableSchemaBuilder.cs

```diff
@@ -40,10 +41,27 @@ public sealed class TableSchemaBuilder : IEnumerable
         public void Add(TableFieldSchema field)
         {
             GaxPreconditions.CheckNotNull(field, nameof(field));
-
             _fields.Add(field);
         }
 
+        /// <summary>
+        /// Modifies the field with <paramref name="fieldName"/> by running the given action.
+        /// </summary>
+        /// <param name="fieldName">The name of the field to be modified. Must not be null.</param>
+        /// <param name="fieldModifier">An action to be performed on the given field.</param>
+        /// <returns>
+        /// This <see cref="TableSchemaBuilder"/> after having applied the modification. For the purposes of method chaining.
+        /// Note: If <paramref name="fieldName"/> or <paramref name="fieldName"/> is null, it will throw an exception.
```

Comment:
Rather than have this as a note in the returns part, let's use an `<exception>` tag, indicating the type of the exception and the cause:

```csharp
/// <exception cref="ArgumentException"><paramref name="fieldName"/> is not present within the schema being built.</exception>
```

Reply:
done